includes/class-wc-gateway-amazon-payments-advanced.php

```diff
@@ -2930,4 +2944,23 @@ public function update_address_details_for_classic( $payload, $checkout_session_
 
 		return $payload;
 	}
+
+	/**
+	 * Get the estimated order amount from the cart totals.
+	 *
+	 * @return string
+	 */
+	private static function get_estimated_order_amount() {
+
+		if ( is_null( WC()->cart ) ) {
```

Comment:
I'd prefer to use `null === WC()->cart` instead. It's more consistent with previous PRs.